### DISCUSSION TITLE
sharing for facebook and pinterest

### DISCUSSION THREAD

**Discussion by DigitalTrading** [openstore-ecommerce/OpenStore#296] [Q&A]
I want to add to each product the ability to share products and images on facebook and pinterest. We used to use addthis, however that has now been retired.
Does anyone have any thoughts on how this can now be done.
Thanks

**Comment by DNNMonster** [comment]
I don't have any suggestions on a replacement widget for social but there are a few out there.   You should be able to modify or create a custom os_FrontOfficePageHeader and ProductDisplayDetail to achieve this.   Usually it's just a couple lines of additional code to get things going.
